content/references/protocol-reference/transactions/transaction-types/accountdelete.md

```diff
@@ -26,6 +26,7 @@ An AccountDelete transaction deletes an [account](accountroot.html) and any obje
     "Flags": 2147483648
 }
 ```
+[Query example transaction. >](https://xrpl.org/websocket-api-tool.html?server=wss%3A%2F%2Fxrplcluster.com%2F&req=%7B%22id%22%3A%22example_AccountDelete%22%2C%22command%22%3A%22tx%22%2C%22transaction%22%3A%221AF19BF9717DA0B05A3BFC5007873E7743BA54C0311CCCCC60776AAEAC5C4635%22%2C%22binary%22%3Afalse%7D)
```

Comment:
Two things, for all of the example buttons here:

1. Please add a blank line between the end of the code sample and this link. (Some Markdown parsers do weird things if you don't have it.)
2. Since these docs are on xrpl.org, we should use a relative link rather than a full URL. That way, it (a) won't display as an "external" link, (b) will go to the "local" version of the page if you view this on staging or if this link gets ported unchanged to another language—rather than always going to the English, production site version of the tool.

```suggestion

[Query example transaction. >](websocket-api-tool.html?server=wss%3A%2F%2Fxrplcluster.com%2F&req=%7B%22id%22%3A%22example_AccountDelete%22%2C%22command%22%3A%22tx%22%2C%22transaction%22%3A%221AF19BF9717DA0B05A3BFC5007873E7743BA54C0311CCCCC60776AAEAC5C4635%22%2C%22binary%22%3Afalse%7D)
```

Reply:
Changed all URLs to relative links.